src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,33 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
+
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    @include square($sideSquare);
+
+    border: 1px solid #000;
+    background-color: #eee;
+    font-size: 30px;
+
+    cursor: pointer;
+    transition-duration: 0.5s;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+      transition-duration: 0.5s;
```

Comment:
you can remove transition-duration from hover, you already have it in &__day selector